src/SlackGhost.ts

```diff
@@ -87,7 +88,7 @@ export class SlackGhost {
         };
     }
 
-    public async update(message: {user_id?: string, user?: string}, room: BridgedRoom) {
+    public async update(message: {user_id?: string, user?: string}, room: BridgedRoom, config: IConfig) {
```

Comment:
I recommend passing in the config in the constructor, and storing the suffix in a property rather than passing it into the update method.

Reply:
That was my original strategy but it became messy when I had to incorporate the tests. I could try doing it again though

Reply:
In my opinion it's a good thing to not have the config on the SlackGhost as long as only `update` uses it.

Reply:
Would still be nice to pass the suffix into the constructor then rather than the update method, imo